OBJ_TO_STL/src/OBJToSTL_Writer.cpp

```diff
@@ -0,0 +1,40 @@
+#include <fstream>
+#include <iostream>
+#include "../headers/OBJToSTL_Writer.h"
+
+OBJToSTL_Writer::OBJToSTL_Writer()
+{
+}
+OBJToSTL_Writer::~OBJToSTL_Writer()
+{
+}
+
+// Reads and writes stl fileinto txt.
+
+Triangulations OBJToSTL_Writer::objToStl(const std::string &filePathName, Triangulations &triangulation)
+{
+    std::ofstream outputFile(filePathName);
```

Comment:
Try to create function with not more than 10 - 15 lines of code. Divide it into sub functions.